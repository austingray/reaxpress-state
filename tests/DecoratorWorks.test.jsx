import React from 'react';
import renderer from 'react-test-renderer';
import TestComponent from './Components/TestComponent';

test('Decorator works', () => {
  const testComponent = renderer.create(<TestComponent />);
  const tree = testComponent.toJSON();
  expect(tree).toMatchSnapshot();
});
