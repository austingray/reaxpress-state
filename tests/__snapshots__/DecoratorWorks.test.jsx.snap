// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Decorator works 1`] = `
<div>
  no color set
</div>
`;
